### DISCUSSION TITLE
Event Trigger with ddl_command_end

### DISCUSSION THREAD

**Discussion by bangtoy** [supabase/supabase#15551] [Questions] [answered]
Anyone Know how to Add EVENT TRIGGER in postgres Supabase ?
i want to monitoring table modification with ddl_command_end, but there is an error because it need superadmin permission.
the Query is like this :

CREATE EVENT TRIGGER table_modification_trigger
ON ddl_command_end
EXECUTE FUNCTION public.handle_table_modification();

**Comment by steve-chavez** [comment, marked as answer]
@bangtoy Post your request here https://github.com/orgs/supabase/discussions/9314. There are demands for superuser capabilities there.
